### ISSUE TITLE
Should deleting remote branch use '--no-verify'?

### ISSUE THREAD

**Issue by angerborn** [magit/magit#3861]
When trying to delete a remote branch via the `x` command, the issued command is something like `git push -v remote :branch` which fails if the `pre-push` hook fails for whatever reasons.

Would it be reasonable to automatically use `--no-verify` when deleting a remote branch?

**Comment by tarsius** [comment]
The `pre-push` hook can be used to prevent deleting a remote branch. So I think you should instead update your hooks to take this into account:

```
           refs/heads/master 67890 refs/heads/foreign 12345

       although the full, 40-character SHA-1s would be supplied. If the
       foreign ref does not yet exist the <remote SHA-1> will be 40 0. If a
       ref is to be deleted, the <local ref> will be supplied as (delete) and
       the <local SHA-1> will be 40 0.
```

**Comment by angerborn** [comment]
Sure, perhaps it is more likely that someone ends up annoyed by deleting something they shouldn't than someone not being able to delete because of the pre-push hook.

Thanks for the answer, I'll update my hook instead.

**Comment by tarsius** [comment]
Something was implemented see #4042.
